integration_tests/Makefile

```diff
@@ -1,18 +1,9 @@
-.PHONY: test-setup test egg-info pull-images provd-test provd
-
 PROVD_DIR ?= ../
 
-
-test-setup: egg-info pull-images provd
-
-pull-images:
-	docker pull wazopbx/wait
-	docker pull trinitronx/python-simplehttpserver
-	docker pull wazopbx/wazo-auth-mock
+test-setup: egg-info provd provd-test
 
 provd:
 	docker build --pull -t wazopbx/wazo-provd $(PROVD_DIR)
-	$(MAKE) provd-test
```

Comment:
Any reason why removing this line? We have this logic in some other services too that you didn't remove. (wazo-chatd)
Since this Makefile is related to tests, I'm not shocked if when we write `make provd` it makes a `provd-test` too. But I don't have strong opinon. 
I have only a strong opinion to do same thing everywhere (the most possible)

Reply:
fixed, I removed the one in wazo-chatd too